sdk/servicebus/azure-servicebus/tests/async_tests/test_sb_client_async.py

```diff
@@ -4,20 +4,28 @@
 # license information.
 #--------------------------------------------------------------------------
 
-
 import logging
 import time
+import asyncio
 import pytest
-
-from azure.core.credentials import AzureSasCredential, AzureNamedKeyCredential
+import hmac
+import hashlib
+import base64
+try:
+    from urllib.parse import quote as url_parse_quote
+except ImportError:
```

Comment:
Do we need this anymore? url2pathname is now under [`urllib.request`](https://docs.python.org/3.7/library/urllib.request.html#urllib.request.pathname2url) .